generate-docs/scripts/preprocessor.ts

```diff
@@ -124,6 +109,53 @@ tryCatch(async () => {
         dtsBuilder.extractDtsSection(definitions, "Begin Word APIs", "End Word APIs")
     );
 
+    // ----
+    // Process Custom Functions d.ts
+    // ----
+    const dtsForCfs = "https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/master/types/custom-functions-runtime/index.d.ts";
```

Comment:
Are we not going to have options for local CF file reading? It'll be nice for testing.

Reply:
Yep, good suggestion. I've added a prompt about source of CF d.ts file and updated processing logic accordingly.